packages/pilot/src/containers/RecipientDetails/Config/style.css

```
@@ -0,0 +1,47 @@
+@import "former-kit-skin-pagarme/dist/styles/spacing.css";
+@import "former-kit-skin-pagarme/dist/styles/colors.css";
+
+.paddingTop > div {
+  padding: 24px 0 0;
+}
+
+.alignItems {
+  display: flex;
+  align-items: center;
+}
+
+.marginLeft {
+  margin-left: var(--spacing-small);
+}
+
+.marginBottomforModal {
+  margin-bottom: var(--spacing-large);
+}
+
+.modalSubtitles {
+  color: var(--color-light-steel-100);
+  text-transform: uppercase;
+}
+
+.justifyContent {
+  display: block;
+  justify-content: flex-end;
+  margin-top: 102px;
+}
+
+.imgSize {
+  width: 250px;
+}
+
+.modalWidth {
+  width: 800px;
+}
+
+.modalHeight {
+  overflow-y: auto;
+  height: 600px;
+}
+
+.marginBottom {
+  margin-bottom: 20px;
```

O idel para estas margens de `20px` é mudar para `24px`(`var(--spacing-large)`) para ficar dentro do nosso design system.
```suggestion
  margin-bottom: var(--spacing-large);
```